proto/runner.proto

```diff
@@ -94,6 +94,10 @@ message RunRequest {
 
   // Max time before run should be canceled automatically.
   string timeout = 12;
+
+  // Exec properties to apply to the run request.
+  repeated build.bazel.remote.execution.v2.Platform.Property exec_properties =
```

Comment:
I kind of preferred having this as a `map<string, string>` that you had before (I think?) because it matches how exec properties are specified in bazel.

Reply:
err, nevermind, this is an internal API so it doesn't matter too much.